winlog.go

```diff
@@ -87,6 +88,18 @@ func (self *WinLogWatcher) SubscribeFromNow(channel, query string) error {
 	return self.subscribeWithoutBookmark(channel, query, EvtSubscribeToFutureEvents)
 }
 
+var curKey uint32
+var wrappers = make(map[uint32]*LogEventCallbackWrapper)
+var wrappersMutex sync.RWMutex
+
+func newEventCallbackWrapper(watcher *WinLogWatcher, channel string) uint32 {
```

Comment:
Here we probably should add a name to return value to simplify understanding, like 
```
func newEventCallbackWrapper(watcher *WinLogWatcher, channel string) (wrapperKey uint32)
```